service_restaurant/src/main/kotlin/org/thechance/service_restaurant/domain/gateway/RestaurantGateway.kt

```diff
@@ -28,11 +26,7 @@ interface RestaurantGateway {
     suspend fun deleteRestaurant(restaurantId: String): Boolean
     suspend fun deleteCategoriesInRestaurant(restaurantId: String, categoryIds: List<String>): Boolean
     suspend fun deleteCuisinesInRestaurant(restaurantId: String, cuisineIds: List<String>): Boolean
+    suspend fun deleteCuisinesFromRestaurant(restaurantId: String, cuisineIds: List<String>): Boolean
```

Comment:
what is the difference between these two functions, they seem do the same behavior?

Reply:
You are right I need to delete one that unused